[docs] Document new scroll actions (closes #6072) (#6086)
DevExpress/testcafe: docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

### Scroll an Element into View

> Since TestCafe scrolls to reach items that are on the page but not on-screen, there is usually no need to manually scroll the page.

Author: > As such, very few test scenarios require explicit scroll directives.

Too wordy.

```suggestion
When TestCafe interacts with an off-screen DOM element, it scrolls that element into view. There is usually no need to use the `scroll` action.
```

> Since TestCafe scrolls to reach items that are on the page but not on-screen, there is usually no need to manually scroll the page.

Reviewer: Sounds good to me, except I'd throw out the word 'explicit', it is unnecessary in this particular case.
```suggestion
When TestCafe interacts with an off-screen DOM element, it scrolls that element into view. There is usually no need to use the `scroll` action.
```

> When TestCafe interacts with an off-screen DOM element, it scrolls that element into view. There is usually no need to use the `scroll` action.

Reviewer 2: ```suggestion
When TestCafe interacts with an off-screen DOM element, it scrolls that element into view. You do not need to use the `scroll` action.
```

> You can use any action (for example, [hover](#hover)) to scroll towards the desired part of the page.  

Author: `Reveal` doesn't work for me in this context.

```suggestion
Interact with a page element (for example, [hover](#hover)) to scroll it into view.
```

> You can use any action (for example, [hover](#hover)) to scroll towards the desired part of the page.  

Reviewer: lgtm

> If you specifically need to scroll the page without any action, use one of the following methods:

Author: `Disrupt` doesn't work here imo.

```suggestion
To scroll the page without interacting with an element, use one of the following methods:
```
